### PR TITLE
Small cleanup

### PR DESCRIPTION
**Pull request by feross**

### CODE REVIEW

**Review by bcomnes — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -3,10 +3,11 @@ module.exports.cli = require('./bin/cmd')
 module.exports.linter = Linter
 
 var deglob = require('deglob')
-var homeOrTmp = require('home-or-tmp')
 var path = require('path')
 var pkgConf = require('pkg-conf')
 
+var HOME_OR_TMP = os.homedir() || os.tmpdir()
```
Comment (Flet) [inline index.js]: os not defined?

Comment (bcomnes) [inline index.js]: Oh yeah good point.

Comment (feross) [inline index.js]: Just added it ;)

Comment (bcomnes) [inline index.js]: Not using a module for this is better here given the obvious nature of this directory selection.

**Review by Flet — APPROVED** [review]

**Comment by feross** [comment]
Finally, tests are passing 👍

**Comment by feross** [comment]
7.0.0

**Pull request merged**